portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSites.macro

```diff
@@ -72,6 +72,38 @@ definition {
 		}
 	}
 
+	@summary = "Default summary"
+	macro viewPageMetric(metricName = null,metricValue = null,pageName = null) {
```

Comment:
Hey @joshitagaki, I think you could change the name of this macro so that it is clear that it checks the metrics of the pages in the list. I think with the current name, someone would think that the macro would work after accessing the page details too (`ACCards.viewMetricValue`)